### DISCUSSION TITLE
1984 개발일정 23-04 5week

### DISCUSSION THREAD

**Discussion by fkdl0048** [The-Day-In-1984/1984_Project#105] [General]
Do It!

**Comment by fkdl0048** [comment]
- #41 
- #48 

이거 두 가지는 레퍼런스 기획 나오는 대로 기능만 제작할 것 같고

회의 때 나온 시스템 구성도에 따른 핵심 시스템먼저 만들 것 같아~

**Comment by kyoonge** [comment]
이번주 평일에 쏟을 수 있는 시간이 별로 없어서ㅠㅜ
주말동안 정면 -> 측면 사다리로 바꿔놓을게!

**Comment by ccmuk58** [comment]
- #104 
위처럼 엑셀 시트가 여러개가 되면서 json 구조가 조금 바뀌었더라고
이거에 맞춰서 기존 있던 변환 기능 개선 예정이용

**Comment by aminseo1023** [comment]
이번주 기획은 날짜별 미션 정하기 완료하겠습니다~!
